### ISSUE TITLE
FastDeploy预编译库中的一键自动压缩工具无法使用

### ISSUE THREAD

**Issue by Taichipeace** [PaddlePaddle/FastDeploy#1286]
*********************************************
温馨提示：根据社区不完全统计，按照模板提问，可以加快回复和解决问题的速度
*********************************************

## 环境

- 【FastDeploy版本】： fastdeploy-gpu/cpu-python==1.0.3 以及develop版本都有这个问题
          预编译库是严格按照这个文档做的：https://github.com/PaddlePaddle/FastDeploy/blob/develop/docs/cn/build_and_install/download_prebuilt_libraries.md
          
- 【系统平台】: Linux x64(Ubuntu 18.04)  aistudio 和 VMware虚拟机里都有这个问题
- 【硬件】： aistudio的16G和32G的GPU都有这个问题
- 【编译语言】： Python3.7
- PaddlePaddle 2.4.0
- PaddleSlim 2.4.0

## 问题日志及出现问题的操作流程
- 严格按照这个文档操作的：https://github.com/PaddlePaddle/FastDeploy/tree/develop/tools/common_tools/auto_compression
日志：
[32m[02-08 14:55:39 MainThread @logger.py:242][0m Argv: /opt/conda/envs/python35-paddle120-env/bin/fastdeploy compress --config_path=./configs/detection/yolov5s_quant.yaml --method=PTQ --save_dir=./yolov5s_ptq_model/
[33m[02-08 14:55:39 MainThread @utils.py:79][0m [5m[33mWRN[0m paddlepaddle version: 2.4.0. The dynamic graph version of PARL is under development, not fully tested and supported
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/parl/remote/communication.py:38: FutureWarning: 'pyarrow.default_serialization_context' is deprecated as of 2.0.0 and will be removed in a future version. Use pickle or the pyarrow IPC functionality instead.
  context = pyarrow.default_serialization_context()
WARNING:root:cannot import name 'layers' from 'parl' (/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/parl/__init__.py)
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/__init__.py:107: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import MutableMapping
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/rcsetup.py:20: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable, Mapping
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/colors.py:53: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sized
Welcome to use FastDeploy Auto Compression Toolkit!
-----------  Running Arguments -----------
 Distillation:
	 alpha: 1.0
	 loss: soft_label
 Global:
	 format: onnx
	 input_list: ['x2paddle_images']
	 model_dir: ./model_exported/yolov5s.onnx
	 model_filename: model.pdmodel
	 params_filename: model.pdiparams
	 ptq_image_path: ./dataset/COCO_val_320
	 ptq_preprocess: yolo_image_preprocess
	 qat_batch_size: 1
	 qat_image_path: ./dataset/COCO_train_320
	 qat_preprocess: yolo_image_preprocess
 PTQ:
	 calibration_method: avg
	 skip_tensor_list: None
 QuantAware:
	 activation_quantize_type: moving_average_abs_max
	 onnx_format: True
	 quantize_op_types: ['conv2d', 'depthwise_conv2d']
	 use_pact: True
 TrainConfig:
	 learning_rate: 1e-05
	 optimizer_builder:
		 optimizer:
			 type: SGD
		 weight_decay: 4e-05
	 target_metric: 0.365
	 train_iter: 3000
------------------------------------------
2023-02-08 14:55:40,318-INFO: Now translating model from onnx to paddle.
2023-02-08 14:55:40,318-WARNING: __init__() missing 1 required positional argument: 'enable_onnx_checker'
2023-02-08 14:55:40,319-ERROR: x2paddle threw an exception, you can ask for help at: https://github.com/PaddlePaddle/X2Paddle/issues
   
截图：
![c6c40e9aab5d44cf79b4e1aef496cc5](https://user-images.githubusercontent.com/94025121/217786462-332eda2d-71b6-4ea3-b6f4-7d27e7c2b8cc.png)

![9afddbc4bfcca51216e71c3a8935118](https://user-images.githubusercontent.com/94025121/217787033-9f4997f8-339d-48c7-84b0-28c29fac9799.png)

- 换用自己的模型也不行
   错误提示2：
![8a30c700b5652e991e3f5db33f237b4](https://user-images.githubusercontent.com/94025121/217786099-a99ca118-126a-4737-8bb6-139c1e418925.png)

个人感觉是预编译库里面的x2paddle工具有问题。

**Comment by wjj19950828** [comment]
@Taichipeace 提供一下ONNX模型，正常使用X2Paddle可以转吗
```shell
 x2paddle -f onnx -m model.onnx -s pd_model
```

**Comment by Taichipeace** [comment]
> @Taichipeace 提供一下ONNX模型，正常使用X2Paddle可以转吗
> 
> ```shell
>  x2paddle -f onnx -m model.onnx -s pd_model
> ```

onnx模型就是官方文档里给的这个：
# 下载yolov5.onnx
wget https://paddle-slim-models.bj.bcebos.com/act/yolov5s.onnx

直接用x2paddle可以转换。

**Comment by Taichipeace** [comment]
这意思是已经找到原因并且修复了？
就等下次合并，然后等发布新版本就可以了？

**Comment by wjj19950828** [comment]
@Taichipeace 等X2Paddle这个PR合入就可以了，我验证了没问题

**Comment by wjj19950828** [comment]
@Taichipeace hi，目前问题已经修复，安装最新版本的X2Paddle，输入下列命令
```shell
## 多pip uninstall几次，确保老版本x2paddle卸载干净
pip uninstall x2paddle
## 安装develop版本
git clone https://github.com/PaddlePaddle/X2Paddle.git
cd X2Paddle
python setup.py install
```

然后再使用FD的auto compress命令

**Comment by Taichipeace** [comment]
好的，非常感谢！
